### ISSUE TITLE
[Proposal] Убрать АФК геймплей у священника и культа

### ISSUE THREAD

**Issue by simb11** [TauCetiStation/TauCetiClassic#10082]
#### Подробное описание проблемы
Убрать унылое афк при использовании алтаря священника и культа. Уменьшить количество времени, нужного для проведения ритуала.
Как предложение, можно сделать так, чтобы продолжительность ритуала зависела от количества людей у алтаря.

**Comment by DarkMinikin** [comment]
Это баланс, прикинь, святоша что после каждой секунды будет спавнить боргов

**Comment by simb11** [comment]
нет, это не баланс, баланс это когда эти заклинания стоят нормальное количество фавора, а не когда кастуются они по 5 минут
